jetbrains: 2024.2.2 -> 2024.2.3

ners opened the pull request:
## Description of changes

## Things done

Ran the update script.

jetbrains.aqua: 2024.2 -> 2024.2.1
jetbrains.goland: 2024.2.2 -> 2024.2.3
jetbrains.idea-community: 2024.2.2 -> 2024.2.3
jetbrains.idea-ultimate: 2024.2.2 -> 2024.2.3
jetbrains.phpstorm: 2024.2.1 -> 2024.2.3
jetbrains.pycharm-community: 2024.2.2 -> 2024.2.3
jetbrains.pycharm-professional: 2024.2.2 -> 2024.2.3
jetbrains.rider: 2024.2.4 -> 2024.2.5
jetbrains.ruby-mine: 2024.2.2 -> 2024.2.3
jetbrains.rust-rover: 2024.2.1 -> 2024.2.2
jetbrains.webstorm: 2024.2.2 -> 2024.2.3

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

uku3lig commented:
hello, intellij idea 2024.2.3 and rustrover 2024.2.2 released since you made your pr, would it be possible to add them or should a separate pr be made?

ners commented:
@uku3lig thanks for the heads-up, re-updated.

ners committed:
jetbrains: 2024.2.2 -> 2024.2.3

jetbrains.aqua: 2024.2 -> 2024.2.1
jetbrains.goland: 2024.2.2 -> 2024.2.3
jetbrains.idea-community: 2024.2.2 -> 2024.2.3
jetbrains.idea-ultimate: 2024.2.2 -> 2024.2.3
jetbrains.phpstorm: 2024.2.1 -> 2024.2.3
jetbrains.pycharm-community: 2024.2.2 -> 2024.2.3
jetbrains.pycharm-professional: 2024.2.2 -> 2024.2.3
jetbrains.rider: 2024.2.4 -> 2024.2.5
jetbrains.ruby-mine: 2024.2.2 -> 2024.2.3
jetbrains.rust-rover: 2024.2.1 -> 2024.2.2
jetbrains.webstorm: 2024.2.2 -> 2024.2.3

ners committed:
jetbrains.plugins: update

uku3lig reviewed (approved):
idea ultimate seems to work fine
